### DISCUSSION TITLE
Welcome to GlobalPlatformPro Discussions!

### DISCUSSION THREAD

**Discussion by martinpaljak** [martinpaljak/GlobalPlatformPro#286] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FloFlex** [comment]
Hello Martin, 

I try to understand how GP secure channels work. You code helps me to get a better approach of the GP specifications.

Thanks for your great job.
